### PR TITLE
Add JWT docs to NodeJS SDK reference

### PR DESCRIPTION
**Pull request by nkshah2**
## Summary of change
Adds function docs for the JWT recipe to NodeJS SDK reference docs

## Related issues
- https://github.com/supertokens/for-zenhub/issues/16

## Related pull requests
- https://github.com/supertokens/supertokens-node/pull/181

### CODE REVIEW

File: v2/nodejs/jwt/init.mdx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+id: init
+title: init
+hide_title: true
+---
+
+```jsx
+JWTRecipe.init({
```
Comment (rishabhpoddar) [inline v2/nodejs/jwt/init.mdx:8]: Why is this called JWTRecipe here? No other recipe has the word Recipe postfix to it.

File: v2/nodejs/jwt/createJWT.mdx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+id: createJWT
+title: createJWT
+hide_title: true
+---
+
+## `createJWT(payload, validity)`
```
Comment (rishabhpoddar) [inline v2/nodejs/jwt/createJWT.mdx:7]: Should be validitySeconds here and below too?

File: v2/nodejs/jwt/createJWT.mdx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+### Parameters
+- `payload`
```
Comment (rishabhpoddar) [inline v2/nodejs/jwt/createJWT.mdx:10]: Shouldn't payload be optional?

File: v2/nodejs/sidebars.js [hunk 4]
```diff
@@ -140,5 +140,14 @@ module.exports = {
         },
       ],
     },
+    {
+      type: 'category',
+      label: 'JWT',
+      items: [
+        "jwt/init",
+        "jwt/createJWT",
+        "jwt/getJWKS",
```
Comment (rishabhpoddar) [inline v2/nodejs/sidebars.js:149]: Need to also add override docs

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

File: v2/nodejs/jwt/createJWT.mdx (hunk 2 above)
Comment (nkshah2) [reply v2/nodejs/jwt/createJWT.mdx]: The signature had `validity` as a parameter in the type declaration, but `validitySeconds` does make more sense. Made the change here and in the NodeJS SDK PR.

File: v2/nodejs/jwt/init.mdx (hunk 1 above)
Comment (nkshah2) [reply v2/nodejs/jwt/init.mdx]: Ive been using JWTRecipe in the NodeJS tests, just defaulted to that here. Changed it to `JWT`

File: v2/nodejs/sidebars.js (hunk 4 above)
Comment (nkshah2) [reply v2/nodejs/sidebars.js:149]: Done

File: v2/nodejs/jwt/createJWT.mdx (hunk 3 above)
Comment (nkshah2) [reply v2/nodejs/jwt/createJWT.mdx]: So in the interface it isnt optional but in hindsight it makes to allow people to create JWT without passing custom payloads. Changed it here (as well as the function override docs) and made changes in the NodeJS PR

**Review by rishabhpoddar — APPROVED** [review]

**Pull request merged**